common/lib/xmodule/xmodule/course_module.py

```diff
@@ -1183,20 +1183,25 @@ def grading_context(self):
 
 
         """
-
+        try:
+            module = getattr(self, '_xmodule', None)
+            if not module:
+                module = self
+        except UndefinedContext:
+            module = self
```

Comment:
The shim code to make XModules behave like XBlocks should be doing this mapping already for you. Also, `module` doesn't appear to be used in the following code. Is this change needed anymore?